src/codegen/cppcg.cpp

```diff
@@ -1226,6 +1227,7 @@ void CppCodeGenerator::GenClassDeclaration(PObjectBase class_obj, bool use_enum,
 	// protected
 	m_header->WriteLn( wxT( "protected:" ) );
 	m_header->Indent();
+    m_header->WriteLn( wxString::Format( wxT( "void Create();\n" ) ) );
```

Comment:
Like mentioned in the associated issue, this name is kind of misleading because it is already used by wxWidgets for two-step creation